sqlmesh/__init__.py

```diff
@@ -75,6 +88,10 @@ def get(cls) -> RuntimeEnv:
 
         if debug_mode_enabled():
             return RuntimeEnv.DEBUGGER
+
+        if is_cicd_environment() or not is_interactive_environment():
```

Comment:
Now that we have CI value should we assign it instead on `is_cicd_environment`? Do we need both values?